modules/mvt/test/mvt-loader.bench.js

```diff
@@ -0,0 +1,50 @@
+import {MVTLoader} from '@loaders.gl/mvt';
+import {setLoaderOptions, fetchFile, parse} from '@loaders.gl/core';
+import {geojsonToBinary} from '@loaders.gl/gis';
+
+setLoaderOptions({
+  _workerType: 'test'
```

Comment:
This sets it globally for all bench marks. Better to set it in each call.

Reply:
Done